CryptoTechReminderSystem.Test/RemindLateDevelopersTests.cs

```diff
@@ -1,84 +1,25 @@
 using System;
 using System.Collections.Generic;
 using CryptoTechReminderSystem.Boundary;
+using CryptoTechReminderSystem.Test.TestDouble;
 using CryptoTechReminderSystem.UseCase;
 using FluentAssertions;
 using NUnit.Framework;
 
 namespace CryptoTechReminderSystem.Test
 {
-    public class RemindLateDevelopersTests
+    public partial class RemindLateDevelopersTests
```

Comment:
Should this be a `partial` class?

Reply:
The `partial` keyword can be removed.

Reply:
Fixed in https://github.com/chubberlisk/crypto_tech/pull/17/commits/baafa2ce54aaa3e9472c571cf71a9025970ca111.